common/commands/config.go

```diff
@@ -3,26 +3,23 @@ package commands
 import (
```

Comment:
Please consider adding some unit tests in `config_test.go` that cover some of the new web logic code